.github/workflows/confluence-daily-dte.yaml

```diff
@@ -0,0 +1,70 @@
+name: Testops Confluence Update
```

Comment:
Can we add Desktop here to the name... I just realized that if not we may see two actions with same name which may be confusing :) 
<img width="275" height="490" alt="Screenshot 2025-11-12 at 17 52 47" src="https://github.com/user-attachments/assets/845e5b6b-08c3-43da-ac0e-fa1ce652f149" />